src/steps/containers.rs

```diff
@@ -24,11 +24,10 @@ fn list_containers(crt: &Path) -> Result<Vec<String>> {
     );
     let output = Command::new(crt)
         .args(["image", "ls", "--format", "{{.Repository}}:{{.Tag}}"])
-        .output()?;
-    let output_str = String::from_utf8(output.stdout)?;
+        .output_checked_with_utf8(|_| Ok(()))?;

```

Comment:
NB: This ignores the exit code.

Reply:
This function uses the closure passed in (here, `|_| Ok(())`) to determine if the command was "completed successfully." By default, it uses the `ExitStatus::success()` for this; you can see that in the implementation of `output_checked` here:

https://github.com/9999years/topgrade/blob/3c298aa4d039c9ddf7932cf55d65e0ccebb5e658/src/command.rs#L89

Reply:
Okay, got it. Thanks for the summary. Your first comment confused me a bit tbh.